### PR TITLE
GLUE-5469 Expand rule to work with all types of bridges

### PR DESCRIPTION
**Pull request by AsonUnique**
#### Overview:
 - Jira ticket: https://spryker.atlassian.net/browse/GLUE-5469

### CODE REVIEW

**Comment by dereuromark** [comment]
At some point these kind of architecture rules must be ported to arch. sniffer instead of code sniffer in the future :)

File: Spryker/Sniffs/DependencyProvider/DependencyNotInBridgeReturnedSniff.php [hunk 1]
```diff
@@ -78,22 +74,27 @@ protected function isCoreProvider(File $phpCsFile): bool
...
-    protected function isFacadeNotInBridgeReturned(File $phpCsFile, int $stackPointer): bool
+    protected function assertDependencyNotInBridgeReturned(File $phpCsFile, int $stackPointer): void
     {
         $tokens = $phpCsFile->getTokens();
         $returnPointer = (int)$phpCsFile->findNext(T_RETURN, $stackPointer);
         $endOfLinePointer = $phpCsFile->findEndOfStatement($returnPointer);
 
         $statementTokens = array_slice($tokens, $returnPointer, $endOfLinePointer - $returnPointer);
         $statement = $this->parseTokensContent($statementTokens);
+        $regExp = sprintf(
```
Comment (gerner-spryker) [inline Spryker/Sniffs/DependencyProvider/DependencyNotInBridgeReturnedSniff.php]: I think there is 1 exception. When you are accessing your own stuff.

\Spryker\Zed\Search\SearchDependencyProvider::addSearchClient
\Spryker\Client\PriceProduct\PriceProductDependencyProvider::addPriceProductService
\Spryker\Zed\PriceProduct\PriceProductDependencyProvider::addPriceProductService

Comment (AsonUnique) [reply Spryker/Sniffs/DependencyProvider/DependencyNotInBridgeReturnedSniff.php]: Thanks for pointing it out, I've added additional check for such cases.

**Comment by sergeysikachev** [comment]
PR is reverted, since affected modules should be updated with bridges

**Comment by dereuromark** [comment]
We tagged a release here!!! you need to provide a fix here ASAP.

**Comment by dereuromark** [comment]
Please also run
```
composer docs
```
after you fixed it

**Pull request merged**